draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -1125,11 +1138,14 @@ intends to include an HTTP response header like
 they could also publish an HTTPSSVC DNS RRSet like
 
     www.example.com. 3600 IN HTTPSSVC 2 svc.example.net. (
-                                        alpn=h3 port=8003 foo=123 )
+                                        proto=h3 port=8003 foo=123 )
                              HTTPSSVC 3 svc.example.net. (
-                                        alpn=h2 port=8002 foo=123 )
+                                        port=8002 foo=123 )
 
 Where "foo" is a hypothetical future HTTPSSVC and Alt-Svc parameter.
+(Note that unlike the Alt-Svc values, these records also indicate support
+for HTTP/1.1 and HTTP/2 on both ports.  This indication could be removed
```

Comment:
Slight nitpick -- TCP 8003 is not the same point as UDP 8003, so "both ports" might not be applicable.  This RRSET makes assertions about *three* ports.